Makefile

```diff
@@ -3,15 +3,15 @@ init:
 	which brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 	brew bundle
 	pre-commit install
+	rbenv install --skip-existing
+	rbenv exec gem update bundler
+	rbenv exec bundle install
 	clang-format --version | awk '{print $$3}' > scripts/.clang-format-version
 	swiftlint version > scripts/.swiftlint-version
 	
 # installs the tools needed to test various CI tasks locally
 init-ci: init
 	brew bundle --file Brewfile-ci
-	rbenv install --skip-existing
-	rbenv exec gem update bundler
-	rbenv exec bundle install
```

Comment:
This is the only part that wasn't strictly part of reverting the previous changes. I noticed this while looking at the makefile: rbenv stuff should only be done locally, not on ci (see xcode-test.sh and IS_LOCAL_BUILD)

Reply:
Why should `rbenv` stuff not be done in CI?